html/host.html

```diff
@@ -3,66 +3,100 @@
     <meta charset="utf-8"/>
     <link href="/assets/stylesheets/main.css" rel="stylesheet">
     <link href="/assets/stylesheets/host.css" rel="stylesheet">
+    <script type="module">
+      import { 
+        fullActivityDescription, 
+        shortActivityDescription, 
+        libraryModelList, 
+        cclDiseaseActivityUrl 
+      } from '/assets/data/host.js';
+
+      document.addEventListener("DOMContentLoaded", () => {
+        // Host page content init
```

Comment:
You can remove most/all of these comments from the code, IMO.

Also, when commenting in the codebase, please end your comment with `--Chris S. (<current date>)`.  The reasoning for this is that comments commonly go out-of-date, and it can be really hard to tell if a comment is actually still a true description of the code it was remarking about.  Knowing how old the comment is, and being able to find a person to ask about it, can be very helpful.

Reply:
Yeah, these comments were mainly for me as I was actually building out the page, not really necessary in the `master` branch, I agree. I looked through my comments & don't think any of them were crucial - nor was anything too confusing IMO or in need of explanation - so I've removed all comments.